.github/actions/general-lints/action.yaml

```diff
@@ -0,0 +1,56 @@
+name: General Lints
```

Comment:
Note: these new files are all just copy/pasted snippets from the existing "required" workflows. The idea is to move the existing "required" workflow steps into these files so we can skip the entire file/flow if we need to 😄